### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

* `error` -> `info`
* Add unit test
* Remove useless comments

### Applicable Issues

#471

### CODE REVIEW

File: cita-chain/core/src/snapshot/service.rs [hunk 1]
```diff
@@ -42,10 +41,7 @@ use libchain::chain::{get_chain, get_chain_body_height, Chain};
 use filters::PollManager;
 
 /// Number of blocks in an ethash snapshot.
-// make dependent on difficulty incrment divisor?
-//const SNAPSHOT_BLOCKS: u64 = 5000;
 /// Maximum number of blocks allowed in an ethash snapshot.
-//const MAX_SNAPSHOT_BLOCKS: u64 = 30000;
```
Comment (ouwenkg) [inline cita-chain/core/src/snapshot/service.rs]:
```
/// Number of blocks in an ethash snapshot.
/// Maximum number of blocks allowed in an ethash snapshot.
```
These two lines have extra usage?

Comment (kaikai1024) [reply cita-chain/core/src/snapshot/service.rs]: NO

Comment (ouwenkg) [reply cita-chain/core/src/snapshot/service.rs]: if no extra consideration, may be we can del it by the way.

Comment (kaikai1024) [reply cita-chain/core/src/snapshot/service.rs]: done

**Pull request merged**